frappe/templates/styles/standard.css

```diff
@@ -166,3 +166,6 @@ table td div {
 	max-height: 150px;
 }
 
+.print-format li[data-list="bullet"] {
+	list-style-type: disc;
```

Comment:
should only be for `ul li` but this should ideally be fixed in the source.

Reply:
@rmehta I was looking at the code for quill as well, there is no ul, just ol and list-style-type set to disc or numeric. So it works fine on text editor but when we go to print, the container changes hence it looses the css.